src/cr/cube/cubepart.py

```diff
@@ -534,6 +534,11 @@ def description(self):
         """str description of this slice, which it takes from its rows-dimension."""
         return self._rows_dimension.description
 
+    @lazyproperty
+    def has_scale_means(self):
```

Comment:
This sounds very much like legacy. I thought we were going to get rid of all those "old" properties that test each thing separately. What do you need this for? I guess something for the client, since it's not used anywhere in the cube. But _what_ exactly in the client? Maybe it could be done differently from the client, to avoid this (rather obsolete) property?

Reply:
We have a TODO: here https://github.com/Crunch-io/zoom/blob/ee846e34bf4b3859fc99bf05dc37ee854c8ccdb3/exporter/src/cr/exporter/spec.py#L804 that exercise the need of this check to be done in cube. And then the need of this 2 props here

Reply:
Anyway I can get rid of this props and revert the exporter business rule. Let me know what's your opinion on this regard